### ISSUE TITLE
After opening the log file, the CPU usage is too high and the idea interface is stuck

### ISSUE THREAD

**Issue by alona789** [JetBrains/ideolog#183]
The log content is like this：
```
2024-02-26 15:18:02 -updating b6f232f9-a76c-4a6c-bfe0-1ed8b710c9cd=29
2024-02-26 15:18:02 -updating 85258f15-090f-462b-96e0-1424d5c5ed9a=31
2024-02-26 15:18:02 -updating 1a0a1a50-a3dd-425f-bb95-3e2eee5525de=33
2024-02-26 15:18:02 -updating 67c2ac7b-0293-43f7-82ea-6b005a6062b8=35
2024-02-26 15:18:02 -updating fe71e9fb-44db-4897-ad03-3287527d0228=37
2024-02-26 15:18:02 -updating 4393f69b-25f5-4460-95f5-5de67c8e80d9=39
2024-02-26 15:18:02 -updating 05d3b72a-93a9-4e09-9764-2f012f960ed1=41
2024-02-26 15:18:02 -updating 9e03416d-14c4-4f63-a328-762595aa1da6=43
```

idea log：
```
2024-02-28 14:55:08,739 [  28625]   WARN - #c.i.s.ComponentManagerImpl - com.liubs.ff.language.LanguageManager initializer requests com.liubs.ff.persistent.MyPluginSettings instance
2024-02-28 14:55:08,832 [  28718]   WARN - #c.i.o.a.ActionStub - ActionGroup should be registered using <group> tag: id="dynamicActionGroup" class="com.liubs.ff.ui.DynamicIpListGroup"
2024-02-28 14:55:09,548 [  29434]   INFO - #com.jpower8.idea.plugin.statictic.api.StatisticStartupActivity - [Statistic] StatisticStartupActivity Plugin 'Statistic' has been activated for project=demo2
2024-02-28 14:55:09,575 [  29461]   INFO - #o.j.j.b.i.CompilerReferenceIndex - backward reference index version doesn't exist
2024-02-28 14:55:09,593 [  29479]   INFO - #c.j.r.p.c.CodeWithMeCleanup - running activity to cleanup old thin clients... Root path is 'C:\Users\user\AppData\Local\JetBrains'. JvmStartTimeMs=1709103279400
2024-02-28 14:55:09,593 [  29479]   INFO - #c.i.c.b.IsUpToDateCheckStartupActivity - suitable consumer is not found
2024-02-28 14:55:09,594 [  29480]   INFO - #c.j.r.p.c.CodeWithMeCleanup - found 0 Code With Me client system folders to check
2024-02-28 14:55:09,595 [  29481]   INFO - #c.j.r.p.c.CodeWithMeCleanup - found 0 Code With Me client config folders to check
2024-02-28 14:55:09,602 [  29488]   INFO - #c.j.r.p.c.CodeWithMeCleanup - keep only [] major versions
2024-02-28 14:55:09,603 [  29489]   INFO - #c.j.r.p.c.CodeWithMeCleanup - Starting the logs folder cleanup. Path: C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log
2024-02-28 14:55:09,605 [  29491]   INFO - #c.i.r.OsRegistryConfigProvider - Looking for 'versionManagementEnabled' value in [HKLM_64\SOFTWARE\JetBrains\JetBrainsClient\versionManagementEnabled, HKLM_32\SOFTWARE\JetBrains\JetBrainsClient\versionManagementEnabled, HKCU_64\SOFTWARE\JetBrains\JetBrainsClient\versionManagementEnabled, HKCU_32\SOFTWARE\JetBrains\JetBrainsClient\versionManagementEnabled]
2024-02-28 14:55:09,606 [  29492]   INFO - #c.i.r.OsRegistryConfigProvider - OS provided value for 'versionManagementEnabled' is not found
2024-02-28 14:55:09,648 [  29534]   INFO - #c.i.d.WindowsDefenderCheckerActivity - real-time protection: false
2024-02-28 14:55:09,669 [  29555]   INFO - #c.i.w.i.i.GlobalWorkspaceModel - Sync global entities with mutable entity storage
2024-02-28 14:55:09,875 [  29761]   INFO - #c.i.w.i.i.EntitiesOrphanageImpl - Update orphanage. 0 modules added
2024-02-28 14:55:09,875 [  29761]   INFO - #c.i.w.i.i.WorkspaceModelImpl - Project model updated to version 7 in 58 ms: Apply JPS storage (iml files)
2024-02-28 14:55:09,876 [  29762]   INFO - #c.i.w.i.i.EntitiesOrphanageImpl - Update orphanage. 0 modules added
2024-02-28 14:55:09,882 [  29768]   INFO - #c.i.w.i.i.j.s.DelayedProjectSynchronizer$Companion - Workspace model loaded from cache. Syncing real project state into workspace model in 381 ms. Thread[DefaultDispatcher-worker-62,5,main]
2024-02-28 14:55:10,810 [  30696]   INFO - #c.i.p.m.MavenProjectDependenciesModel - Start collecting all modules
2024-02-28 14:55:10,813 [  30699]   INFO - #c.i.p.g.GradleProjectDependenciesModel - Start collecting all modules
2024-02-28 14:55:10,814 [  30700]   INFO - #c.i.p.g.GradleProjectDependenciesModel - Finish collecting all modules, count of found modules: 0
2024-02-28 14:55:10,819 [  30705]   INFO - #c.i.p.m.i.ProjectDependenciesModelImpl - Count of declared packages: null, imported packages: 0
2024-02-28 14:55:10,821 [  30707]   INFO - #c.i.p.m.MavenProjectDependenciesModel - Finish collecting all modules, count of found modules: 1
2024-02-28 14:55:10,824 [  30710]   INFO - #c.i.p.m.i.ProjectDependenciesModelImpl - Count of declared packages: null, imported packages: 117
2024-02-28 14:55:10,911 [  30797]   INFO - #c.i.p.m.i.ProjectDependenciesModelImpl - Count of declared packages: null, imported packages: 182
2024-02-28 14:55:11,819 [  31705]   INFO - #c.i.p.m.ProjectDependenciesModuleAggregator - Supported models after update: [IntellijModuleProjectDependenciesModel, IntellijProjectDependenciesModel, MavenProjectDependenciesModel]
2024-02-28 14:55:11,897 [  31783]   INFO - #c.i.p.m.i.ProjectDependenciesModelImpl - Count of declared packages: 65, imported packages: 182
2024-02-28 14:55:14,417 [  34303]   INFO - STDOUT - 
2024-02-28 14:55:30,782 [  50668]   INFO - #c.i.d.PerformanceWatcherImpl - High memory usage (free 1206 of 8192 MB) while dumping threads to C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log\threadDumps-freeze-20240228-145519-IU-233.14475.28\threadDump-20240228-145530.txt
2024-02-28 14:55:32,781 [  52667]   INFO - #c.i.o.u.LowMemoryWatcher - Low memory signal received: afterGc=false
2024-02-28 14:55:57,925 [  77811]   INFO - #c.i.u.g.s.GistStorageImpl - Cleaning old huge-gists dirs from [C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\caches\huge-gists] ...
2024-02-28 14:55:57,928 [  77814]   INFO - #c.i.u.g.s.GistStorageImpl - Cleaning old huge-gists dirs finished
2024-02-28 14:56:08,152 [  88038]   WARN - #c.i.d.PerformanceWatcherImpl - UI was frozen for 53826ms, details saved to C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log\threadDumps-freeze-20240228-145519-IU-233.14475.28-JComponent.paintToOffscreen-53sec
2024-02-28 14:56:26,739 [ 106625]   WARN - #c.i.d.PerformanceWatcherImpl - UI was frozen for 18542ms, details saved to C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log\threadDumps-freeze-20240228-145613-IU-233.14475.28-EditorPainter$Session.paint-18sec
2024-02-28 14:56:31,800 [ 111686]   INFO - #c.i.d.PerformanceWatcherImpl - High memory usage (free 1582 of 8192 MB) while dumping threads to C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log\threadDumps-freeze-20240228-145631-IU-233.14475.28\threadDump-20240228-145631.txt
2024-02-28 14:56:41,380 [ 121266]   INFO - #c.i.o.u.LowMemoryWatcher - Low memory signal received: afterGc=false
2024-02-28 14:56:45,322 [ 125208]   WARN - #c.i.d.PerformanceWatcherImpl - UI was frozen for 18575ms, details saved to C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2023.3\log\threadDumps-freeze-20240228-145631-IU-233.14475.28-EditorPainter$Session.paint-18sec
2024-02-28 14:56:49,897 [ 129783]   INFO - #c.i.o.u.LowMemoryWatcher - Low memory signal received: afterGc=false

```

**Comment by honzavaclavik** [comment]
I have same problem.

**Comment by Wimpy-Kid** [comment]
I'm stuck with the same issue :(

**Comment by honzavaclavik** [comment]
> I'm stuck with the same issue :(

Disabling the plugin "solves" the pain :)

**Comment by alona789** [comment]
> > I'm stuck with the same issue :(我遇到了同样的问题:(
> 
> Disabling the plugin "solves" the pain :)禁用插件“解决”了痛苦:)

Me too :<

**Comment by brian-thomas-hpe** [comment]
Me too. I had to diassociate .log files in the plugin.  Every .log file no matter how small would cause IntelliJ to hang

**Comment by smirok** [comment]
According to #177, it was fixed in 222.3.2.0
